### DISCUSSION TITLE
How can I set the config locally not globally

### DISCUSSION THREAD

**Discussion by dongho-jung** [mmistakes/minimal-mistakes#2477] [Q&A] [answered]
Hello [this](https://github.com/0xF4D3C0D3/0xF4D3C0D3.github.io) is my jekyll mm repo.

![image](https://user-images.githubusercontent.com/19762154/78549177-eabf5500-783c-11ea-8e14-53ab4221e0b7.png)
and I want to disable line number in only some posts or some blocks. But what I only know is just editing _config.yml however it applies all posts not only some posts. How can I do that? is it possible?

**Comment by iBug** [comment]
[This](https://stackoverflow.com/a/25133352/5958455) is what you're looking for.

**Comment by dongho-jung** [comment]
@iBug thanks for your answer,

I already saw it but as you can see I don't want any highlights in the above examples
so I wrote them like just \``` ... \``` for only pre-formatted alignment not highlighting

and now I'm waiting for applying this new approach but it takes a lot of time

**Comment by dongho-jung** [comment]
I think it won't be updated because I omitted the language in the {% highlight %} tag.
so it looks like that it needs some language to highlight, but I don't want to highlight anything. I just want pre-formatted code block

**Comment by iBug** [comment, marked as answer]
Use `text` language to explicitly disable language highlighting.

**Comment by dongho-jung** [comment]
oooh yeah! I got it! thanks!! wow, I didn't notice it. that's exactly what I looking for.

thank you very much iBug :D
